src/index.js

```diff
@@ -75,7 +72,6 @@ const crudTodos = () => {
     }
   });
 
-  /*  EVENT LISTENERS delete completed todos */
   const removeCompleted = document.getElementById('remove-completed');
   removeCompleted.addEventListener('click', () => {
     removeComleted();
```

Comment:
- [ ] you have the required test but please ensure that they are passing. In the case of your tests, it looks like you have to mock the `localStorage` please make this change. the point of testing is to ensure that your code functions as expected so you need to ensure your tests are always passing.   
![image](https://user-images.githubusercontent.com/18730106/151443754-c61de7c6-29e8-4cbb-8e45-b59b847591b1.png)